src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -223,19 +240,12 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
     const double fStd = arma::stddev(feature, population);
 
     // Print statistics of the given dimension.
-    Log::Info << boost::format(numberFormat)
-        % dim
-        % arma::var(feature, population)
-        % fMean
-        % fStd
-        % arma::median(feature)
-        % fMin
-        % fMax
-        % (fMax - fMin) // range
-        % Skewness(feature, fStd, fMean, population)
-        % Kurtosis(feature, fStd, fMean, population)
-        % StandardError(feature.n_elem, fStd)
-        << endl;
+    Log::Info << string_format(numberFormat,
+        dim, arma::var(feature, population), fMean, fStd,
+        arma::median(feature), fMin, fMax, (fMax - fMin),
+        Skewness(feature, fStd, fMean, population),
+        Kurtosis(feature, fStd, fMean, population),
+        StandardError(feature.n_elem, fStd)) << endl;
```

Comment:
I would just remove the extra `string_format` function and do the output format manually Something like:

```c++
Log::Info << "dim: " << dim
               << " var: " << arma::var(feature, population)
               << ...
```

That way we don't have to backport and maintain another function we only use once.

Reply:
I think if you want to remove that function, I can do as you suggested.

Reply:
But I think this function takes account of the precision of float number printing.

Reply:
You can control that with, e.g., [setprecision](https://en.cppreference.com/w/cpp/io/manip/setprecision) or [setw](https://en.cppreference.com/w/cpp/io/manip/setw).  Personally I think it's okay if we change the output style just a little bit, if it lets us remove an internal implementation. :)